docs/hub/spaces-sdks-docker-argilla.md

```diff
@@ -10,22 +10,31 @@ In the next sections, you'll learn to deploy your own Argilla app and use it for
 
 In this section, you'll learn to deploy an Argilla Space and use it for data annotation and training a sentiment classifier with [SetFit](https://github.com/huggingface/setfit/), an amazing few-shot learning library.
 
-### Duplicate the Argilla Space Template and create your Space
+### Deploy Argilla on Spaces
 
-The easiest way to get started is by [duplicating the Argilla Docker Template](https://huggingface.co/spaces/argilla/argilla-template-space?duplicate=true). You need to define the **Owner** (your personal account or an organization), a **Space name**, and the **Visibility**. To interact with the Argilla app with Python, you need to setup the visibility to `Public`. Once you are all set, click "Duplicate Space". 
+You can deploy Argilla on Spaces with just a few clicks:
+
+<a  href="https://huggingface.co/new-space?template=argilla/argilla-template-space">
+    <img src="https://huggingface.co/datasets/huggingface/badges/raw/main/deploy-to-spaces-lg.svg" />
+</a>
+
+You need to define the **Owner** (your personal account or an organization), a **Space name**, and the **Visibility**. To interact with the Argilla app with Python, you need to setup the visibility to `Public`.
+
+<Tip>
+If you want to customize the title, emojis, and colors of your space, go to "Files and Versions" and edit the metadata of your README.md file.
+</Tip>
 
 <div class="flex justify-center">
 <img src="https://huggingface.co/datasets/huggingface/documentation-images/resolve/main/hub/spaces-argilla-duplicate-space.png"/>
```

Comment:
Maybe here we should show a screenshot of the Spaces creation page

Reply:
I've uploaded a new image and changed the url, I guess once the PR is merged here we're fine: https://huggingface.co/datasets/huggingface/documentation-images/discussions/42

Reply:
That PR Is merged :) the url here is still the same though

Reply:
Now it's changed, sorry!